Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ErrorLogBean.java

```diff
@@ -69,7 +73,12 @@
     private List<LogEventType> eventsList;
 
     public ErrorLogBean() {
-        dropdowns = service.getDropdowns();
+        try {
```

Comment:
Don't line catching exception in the constructor. use an init method with @PostConstruct annotation.